### DISCUSSION TITLE
Including Songs for the Weekend Meeting

### DISCUSSION THREAD

**Discussion by ajtruckle** [ajtruckle/meeting-schedule-assistant-support#98] [Workbook]
Meeting Schedule Assistant allows you to create **custom reports** or use **existing ones** to meet all your **reporting needs**. Here’s an updated version of built-in **Workbook S-140 Full** that includes the **Songs** information for the **Weekend Meeting**.

![Sample Preview](https://github.com/user-attachments/assets/dc1b0357-8087-437d-9bae-fc48d102757f)

### Sample Script File ###
This script can be downloaded as a **compressed (ZIP)** file that includes these essential components:

1. **Template (XSL File)**: This file's name will appear in the **drop-down list** within the **Meeting Editor**.

Download: 
[Workbook-S-140-Full-Songs](https://www.publictalksoftware.co.uk/?sdm_process_download=1&download_id=13963&sdm_ignore_logging=1)

| File        | Filename                       |
|:-------------|:--------------------------------|
| Template  | Workbook-S-140-Full-Songs.xsl |

## Installing the Script File (ZIP) ##
To install the script file, follow these steps:

1. **Open Meeting Editor**: Navigate to the Meeting Editor and click on the "Workbook" tab in the right panel.
2. **Click Import Style**: Press the [Import Style](https://www.publictalksoftware.co.uk/help-msa/mwb-editor-section8.html#Import_Template) button.
3. **Select Workbook File Type**: From the list, choose "Workbook ZIP" as the file type and locate the ZIP file you downloaded.
4. **Click Open**: The script will now appear in the Style drop-down list

**Comment by ajtruckle** [comment]
# Revision History

## Version 1.0 - Released 06.08.2024

- Initial release.

**Comment by MardoDan** [comment]
Podrías compartir una plantilla similar pero que sea de la Reunión de Fin de Semana que incluya las canciones??? Sería basada en la plantilla Workbook-S-140-Weekend Meeting
Muchas gracias.

**Comment by ajtruckle** [comment]
@MardoDan 
If you open the built-in `Workbook-S-140-Weekend Meeting` template and make a copy, then you can simply repeat the same process for adding the songs. Or, as an alternative, make a copy of your new template and comment out the same lines to suppress the display of midweek meeting records.
